model_compression_toolkit/keras/gradient_ptq/training_wrapper.py

```diff
@@ -69,9 +69,15 @@ def gptq_training_wrapper(tg: Graph,
                                               fw_info=fw_info,
                                               gptq_config=gptq_config)
 
-    trainable_weights = get_trainable_parameters(fxp_model,
-                                                 fw_info,
-                                                 add_bias=gptq_config.train_bias)
+    trainable_weights, flp_weights_list, fxp_weights_list = \
+        get_trainable_parameters(fxp_model,
+                                 fw_info,
+                                 add_bias=gptq_config.train_bias)
+
+    if len(compare_points) != len(trainable_weights):
```

Comment:
Where is the check that len(fxp_weights_list)=len(flp_weights_list)=len(compare_points)
Also where is the batch-normalization prior?